tests/testSkyModel.py

```diff
@@ -160,7 +158,7 @@ def test90Deg(self):
         sm = self.sm_mags
         sm.setRaDecMjd(0., 90., mjd, degrees=True, azAlt=True)
         mags = sm.returnMags()
-        for key in mags.dtype.names:
+        for key in mags.keys():
```

Comment:
I tend to put the .keys() in there for my own readability since I can never remember what gets iterated. Good point about Python 3.